### ISSUE TITLE
seems like isol and tfrm need to be annotations

### ISSUE THREAD

**Issue by Krisoyo** [TarkanV/GraphChannelFilter#2]
Addon wouldn't work properly in 2.93, so I had a quick look and got it to work after changing
>isol = bpy.props.BoolProperty()
tfrm = bpy.props.StringProperty()

into
>isol: bpy.props.BoolProperty()
tfrm: bpy.props.StringProperty()

**Comment by TarkanV** [comment]
Yeah thanks for pointing this out, that seems to be caused by a change with recent blender versions. I'll update the main project.

**Comment by TarkanV** [comment]
Fixed.
